### PR TITLE
First Legal Notice for WCC

### PR DESCRIPTION
**Pull request by clickysteve**

### CODE REVIEW

File: legal/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+The Alpha release of the WooCommerce Connect Client (WCC) is for testing purposes, and as such should not be used on production sites. You do so at your own risk.
+
+In order to facilitate the live shipping rate  lookup from the relevant shipping operator (USPS/Canada Post), data must be transferred to their servers from your own, via our intermediary server. An outline of the process is as follows:
+
+* The weight, dimensions, and product IDs of the items in the customer's cart are transferred to the WCC server, along with their complete address.
+* Neither the customer's name, or the name of the products are transferred.
+* This data is not stored persistently on the WCC server.
+* The above data is sent to the shipping operators (along with the postal/zip code of the merchant), who then calculate the shipping cost and provide that in response.
+* Other pieces of data are transferred to the WCC server in addition for statistical purposes, but not passed on to the shipping operators. These include the site's Jetpack token; currency, country, and state setting; locale; and postal code.
```
Comment (allendav) [inline legal/README.md]: I suggest changing `are transferred to the WCC server in addition for statistical purposes` to `are transferred to the WCC server for authentication and statistical purposes`

File: legal/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* TRACKS events are recorded when you add, remove, enable, or disable a shipping service. The info recorded includes your site URL, browser user agent, and IP address.
```
Comment (allendav) [inline legal/README.md]: Will a merchant or dev know what a WordPress.com TRACKS event is? Maybe we should say `The WCC Server receives events when you...`

File: legal/README.md (hunk 1 above)
Comment (pesieminski) [inline legal/README.md]: Sounds good.

File: legal/README.md (hunk 2 above)
Comment (pesieminski) [inline legal/README.md]: Yes, agreed.

**Comment by allendav** [comment]
Moving this out to Beta, since we plan on cutting the Alpha 2 tag today. If this causes concern, please let me know.

**Comment by allendav** [comment]
@clickysteve  @pesieminski - I was just reading https://en.wordpress.com/tos/ and it occurred to me - at the end of it all, instead of this legal notice, should we be linking to updated WordPress.com TOS and [Privacy policy](https://automattic.com/privacy/) docs  instead?

**Pull request closed without merging**